cmd/create.go

```diff
@@ -106,23 +105,24 @@ func newCreateConfig(args []string) createConfig {
 	}
 }
 
-// Prompt the user with value of config members, allowing for interaractive changes.
+// Prompt the user with value of config members, allowing for interactive changes.
 // Skipped if not in an interactive terminal (non-TTY), or if --confirm (agree to
 // all prompts) was not explicitly set.
 func (c createConfig) Prompt() createConfig {
-	name := deriveName(c.Name, c.initConfig.Path)
 	if !interactiveTerminal() || !c.initConfig.Confirm {
 		// Just print the basics if not confirming
 		fmt.Printf("Project path: %v\n", c.initConfig.Path)
-		fmt.Printf("Project name: %v\n", name)
+		fmt.Printf("Function name: %v\n", c.initConfig.Name)
 		fmt.Printf("Runtime: %v\n", c.Runtime)
 		fmt.Printf("Trigger: %v\n", c.Trigger)
 		return c
 	}
+
+	derivedName, derivedPath := deriveNameAndAbsolutePathFromPath(prompt.ForString("Project path", c.initConfig.Path, prompt.WithRequired(true)))
```

Comment:
This `deriveNameAndAbsolutePathFromPath()` function is called in [`newInitConfig()`](https://github.com/boson-project/faas/pull/127/files#diff-46777a3b1ee86d4cfc7d0bef9b3b7ee8R85) which is called by [`newCreateConfig()`](https://github.com/boson-project/faas/pull/127/files#diff-452662b1e58d63509f34fc4bd164905bL103),  so it seems redundant to call it again here. The values for `c.initConfig.Path` and `c.initConfig.Name` should already be set, shouldn't they?

Reply:
This one is for prompt. Yes, the value is there, but user could replace `path` in the prompt, then we need to get the path and name again.